SeaSharpe_CVGS/SeaSharpe_CVGS/Views/Event/Delete.cshtml

```diff
@@ -57,7 +57,7 @@
         @Html.AntiForgeryToken()
 
         <div class="form-actions no-color">
-            <input type="submit" value="Delete" class="btn btn-default" /> |
+            <input type="submit" value="Delete" class="btn btn-default, greenButton" /> |
```

Comment:
Classes shouldn't be delimited by commas. Just use a space.

Instead of making a green button class we can replace btn-default with btn-success. Reference: http://www.w3schools.com/bootstrap/bootstrap_buttons.asp